go.mod

```diff
@@ -1,17 +1,71 @@
 module github.com/brancz/kube-rbac-proxy
 
-go 1.13
+go 1.17
 
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/oklog/run v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/net v0.0.0-20200707034311-ab3426394381
-	gopkg.in/yaml.v2 v2.2.8
+	golang.org/x/net v0.7.0
+	gopkg.in/yaml.v2 v2.3.0
 	k8s.io/api v0.19.2
 	k8s.io/apimachinery v0.19.2
 	k8s.io/apiserver v0.19.2
 	k8s.io/client-go v0.19.2
```

Comment:
The k8s APIs can be upgraded to v0.20.* (like [0.20.15](https://pkg.go.dev/k8s.io/client-go@v0.20.15)) to fix the one remaining MEDIUM vuln - though we don't have to.

Reply:
So are we done here atleast from high and critical vulnerabilities standpoint ?

Reply:
Obviously. If there are 0 CRIT and 0 HIGH vulns, ...